src/widgets/Tabs/PatientSelect.js

```diff
@@ -253,12 +256,16 @@ const PatientSelectComponent = ({
           rowKey={keyExtractor(item)}
           columns={columns}
           onPress={name => {
-            // Only show a spinner when the name doesn't exist in the database, as we need to
-            // send a request to the server to add a name store join.
-            if (UIDatabase.get('Name', name?.id)) {
-              selectPatient(name);
+            if (!name?.isDeceased) {
+              // Only show a spinner when the name doesn't exist in the database, as we need to
```

Comment:
@chastichno Similar concern to previous thing I raised, but I just noticed that when we select a patient we are showing the popup and preventing further page navigation here. I think we should allow view of the patient detail, we should only stop navigation when the app user tries to dispense to deceased user.

Reply:
I think that it's ok to stop at this point, since the purpose of this workflow was to dispense a vaccine. It has been hacked around a fair bit ( at customer's request ) to enable editing and re-editing.. but the purpose here is to find a patient and dispense to them. 

One reason to show the details and allow editing would be to allow the user to bring the patient back from the dead.. which may be valid if they were changed accidentally. Even so.. I'd recommend that we push people to use the normal patient edit.

It would help though, to have an extra column so that you don't click on a dead person. Even better.. why don't we just not show dead people in the search results?